.travis.yml

```diff
@@ -0,0 +1,15 @@
+sudo: required
+language: java
+jdk:
+  - oraclejdk8
+before_cache:
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+cache:
+  directories:
+    - $HOME/.gradle/caches/
+    - $HOME/.gradle/wrapper/
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
+  - pip install --user awscli
+  - eval $(aws ecr get-login --no-include-email --region us-east-1)
```

Comment:
We don't need the last two lines.

Reply:
Those lines are being used to push a docker image to ecr which this project does not have yet.